app/presenters/bulk_exports_presenter.rb

```diff
@@ -16,7 +16,7 @@ def initialize
   private
 
   def init_exported_at_message
-    export_executed_at = DefraRuby::Exporters::BulkExportFile.first&.created_at
+    export_executed_at = DefraRuby::Exporters::BulkExportFile.first&.created_at&.strftime("%-d %B %Y")
```

Comment:
Just to add to that, hadn't meant for the time element to be dropped! The screen reader was fine with that part, and I think it will help users with assessing how 'fresh' the data is.

However I'd stick with using an initializer, it's just you might need to add one that also includes the timestamp (HH:mm should be sufficient) as well date, month and year.

Reply:
Fixed in c67bf02.